README.md

```diff
@@ -151,32 +151,43 @@ target:
 It is worth mentioning that you can use Harbor robot accounts using OCI registries as source or target.
 
 Also, take into account that if you use OCI as the source repository you must specify the list of charts to synchronize
-or a pointer to an [index file](#helm-charts-index-for-oci-based-repositories) in the repository.
+or a pointer to a [remote index file](#remote-index-for-oci-based-repositories) in the repository.
 
-#### Helm Charts Index for OCI-based repositories
+#### Remote index for OCI-based repositories
```

Comment:
What's wrong with calling it Helm Charts index? I understand that's not an official solution and I believe that we make it clear somewhere else, but to me an index needs to indicate what's indexing,